### ISSUE TITLE
Suggestion: Nickname tabcomplete

### ISSUE THREAD

**Issue by skmzarn** [funkydude/BasicChatMods#24]
I know this is a very basic chatmod, and I like it the way it is, but I'm missing this feature from other chat addons. You start typing a nick, and the addon will autocomplete the nick for you - from the guildroster, friendlist, your current group - and if there are alternatives, you will tab between them.

This small addon does just what I want, but it also completes the realm name after the nickname, which is quite irritating, even from your own realm.
https://www.wowinterface.com/downloads/info6761-NickCompleter.html

I also made this thread on MMO to try to get some help: 
https://www.mmo-champion.com/threads/2514596-Nick-Completer-addon-without-realm-name

Thanks for reading!

**Comment by funkydude** [comment]
When you use a command e.g. `/w playername` the Blizz auto complete already shows up.
I assume you already see this, but, what you want is auto complete of a name mid sentence?

**Comment by skmzarn** [comment]
Yes, you are correct. I want to be able to autocomplete nicknames read from guild roster/group/friend lists, by just using TAB, anywhere in the sentence, and without it being a command. Kind of a quality of life feature.
